cli/create_indices.go

```diff
@@ -38,7 +38,20 @@ func (c *createIndices) ShouldRun() bool {
 }
 
 func (c *createIndices) Run() {
-	_, err := c.esClient.CreateIndex(person.Person{})
+	exists, err := c.esClient.IndexExists(person.Person{})
+	if err != nil {
+		c.logger.Println(err)
+		c.exit(1)
```

Comment:
Is exit what we want here? Wouldn't this mean the whole search service process would exit and need to be restarted? I might be wrong but I thought I'd check.